### ISSUE TITLE
CBL iOS library build fails with property synthesis errors in Xcode 6.3

### ISSUE THREAD

**Issue by pegli** [couchbase/couchbase-lite-ios#660]
Yay, another Xcode release with new and exciting compiler defaults!  When I build the CBL iOS Library target in Xcode 6.3 (6D570), I see several errors like this one:

```
error: auto property synthesis will not synthesize property 'isDeletion'; it will be implemented by its
superclass, use @dynamic to acknowledge intention [-Werror,-Wobjc-property-synthesis]
```

If you don't want to change `@property` to `@dynamic` at this time, you can add `-Wno-objc-property-synthesis` to the "Other Warning Flags" section of the build settings.

**Comment by pasin** [comment]
Hello @pegli, I have the same XCODE (6D570), but I don't see the any error when building the library.

What scheme did you select when building the library?

**Comment by pegli** [comment]
I should have mentioned that I'm building from the release/1.0.4 and release/1.0.4-hotfix branches.  The master branch builds fine.

**Comment by snej** [comment]
I fixed Xcode 6.3 build errors on the master branch last month(?), but since 1.0.4 was frozen back in December I didn't go and backport the fixes there...

**Comment by pegli** [comment]
Ok, since those branches are closed, there is no need to fix the build settings.  If anyone else runs into the same problem, a search should bring them to this issue report and the workaround I listed above.
